pdb/pom.xml

```diff
@@ -114,6 +121,8 @@
 			</plugin>
 		</plugins>
 	</build>
-
+	<properties>
+		<start-class>org.cbioportal.pdb_annotation.scripts.PdbAnnotationScriptsPipelineStarter</start-class>
```

Comment:
Instead of tweaking the current pdb module by this way, we should add another module. This turns the current web application into a command line tool where we can only execute a single script.